classes/PodsInit.php

```diff
@@ -150,6 +159,176 @@ public function plugins_loaded() {
 
 		load_plugin_textdomain( 'pods' );
 
+		$this->freemius();
+
+	}
+
+	/**
+	 * Handle Freemius SDK registration.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return \Freemius
+	 */
+	public function freemius() {
+		if ( $this->freemius ) {
+			return $this->freemius;
+		}
+
+		require_once dirname( __DIR__ ) . '/vendor/freemius/wordpress-sdk/start.php';
+
+		try {
+			$this->freemius = fs_dynamic_init( [
+				'id'             => '5347',
+				'slug'           => 'pods',
+				'type'           => 'plugin',
+				'public_key'     => 'pk_737105490825babae220297e18920',
+				'is_premium'     => false,
+				'has_addons'     => true,
+				'has_paid_plans' => false,
+				'menu'           => [
+					'slug'        => 'pods-settings',
+					'contact'     => false,
+					'support'     => false,
+					'affiliation' => false,
+					'account'     => true,
+					'pricing'     => false,
+					'addons'      => true,
+					'parent'      => [
+						'slug' => 'pods',
+					],
+				],
+			] );
+
+			$this->override_freemius_strings();
+
+			add_filter( 'fs_plugins_api', [ $this, 'filter_freemius_plugins_api_data' ], 15 );
+
+			$this->freemius->add_filter( 'templates/add-ons.php', [ $this, 'filter_freemius_addons_html' ] );
+			$this->freemius->add_filter( 'download_latest_url', [ $this, 'get_freemius_action_link' ]  );
+
+			/**
+			 * Allow hooking into the Freemius registration after Pods has registered it's own Freemius.
+			 */
+			do_action( 'pods_freemius_init' );
+		} catch ( \Exception $exception ) {
+			return;
+		}
+
+		return $this->freemius;
+	}
+
+	/**
+	 * Override Freemius strings.
+	 */
+	public function override_freemius_strings() {
+		$override_text = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 01e13b438f9b1fb69bc5de6e3fd4cfb2</sub>